open-metadata-implementation/repository-services/repository-services-implementation/src/main/java/org/odpi/openmetadata/repositoryservices/rest/server/OMRSRepositoryRESTServices.java

```diff
@@ -8005,13 +8005,13 @@ private void captureCheckedException(OMRSAPIResponse          response,
     {
         response.setRelatedHTTPCode(error.getReportedHTTPCode());
         response.setExceptionClassName(exceptionClassName);
```

Comment:
Reordering of code properties may cause mistakes in the future and no longer follows consistent ordering.

Reply:
Opened #2874 to track independently. will go with the merge here for now